src/Components/Todo.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+function Todo({ lists, completeTodo }) {
+    const handleClick = id => {
+        completeTodo(id);
+    };
+    return lists.map(function(list) {
```

Comment:
Gave up on the arrow functions?? :D 
This is a better example of destucturing it here;
```
return lists.map({ completed, id, task } => {
```